src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -45,29 +53,81 @@ limitations under the License.
     <attribute side="server" code="0x0016" define="MAX_HEAT_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0BB8" optional="true">max heat setpoint limit</attribute>
     <attribute side="server" code="0x0017" define="MIN_COOL_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0640" optional="true">min cool setpoint limit</attribute>
     <attribute side="server" code="0x0018" define="MAX_COOL_SETPOINT_LIMIT" type="INT16S" min="0x954D" max="0x7FFF" writable="true" default="0x0C80" optional="true">max cool setpoint limit</attribute>
```

Comment:
The 4 min/max setpoint limit attributes need to have "manage" for write, no?

Reply:
OK merged in master again, went over the file again, should be good now.